src/main/io/vtx_smartaudio.c

```diff
@@ -503,7 +501,7 @@ static saCmdQueue_t sa_queue[SA_QSIZE];
 static uint8_t sa_qhead = 0;
 static uint8_t sa_qtail = 0;
 
-#ifdef DPRINTF_SMARTAUDIO
+#ifdef USE_DPRINTF_SMARTAUDIO
```

Comment:
This one stands out, others are `USE_SMARTAUDIO_DPRITNF`

Reply:
Hurr durr, well spotted. 'It was like this when I found it!'